src/core/dfn-panel.js

```diff
@@ -0,0 +1,161 @@
+// Constructs "dfn panels" which show all the local references to a dfn and a
+// self link to the selected dfn. Based on Bikeshed's dfn panels at
+// https://github.com/tabatkins/bikeshed/blob/ef44162c2e/bikeshed/dfnpanels.py
+import { fetchAsset } from "./text-loader.js";
+import { hyperHTML } from "./import-maps.js";
+import { norm } from "./utils.js";
+
+const cssPromise = loadStyle();
+
+export const name = "core/dfn-panel";
+
+export async function run() {
+  const css = await cssPromise;
+  document.head.insertBefore(
+    hyperHTML`<style>${css}</style>`,
+    document.querySelector("link")
+  );
+
+  document.body.addEventListener("click", event => {
+    /** @type {HTMLElement} */
+    const el = event.target;
+    const panel = document.getElementById("dfn-panel");
```

Comment:
Maybe move this to the outer function? Then only needs to do the lookup once?

Reply:
Can't. We remove panel from DOM instead of just hiding it (as we recreate entire content - which is simpler)
This reminds me, we can simplify CSS due to that.

Reply:
Removed the "on" state as if it exists, it's either minimzed (activated) or visible,

Reply:
Made some improvements in https://github.com/w3c/respec/pull/2626/commits/e562d736e3fa3492446988c76c146ef0da214531